includes/application/forwardApprover.php

```diff
@@ -51,7 +51,7 @@
 
 if ( ! isset (  $_POST['app_id'] ) )
 {
-	echo "Sorry, there was a problem.<br>";
+	echo "Sorry, there was a problem, post data.<br>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->